modules/rosa-architecture-local-zones.adoc

```diff
@@ -3,16 +3,16 @@
 // * architecture/rosa-architecture-models.adoc
 :_mod-docs-content-type: REFERENCE
 [id="rosa-architecture-local-zones_{context}"]
-= ROSA architecture with Local Zones
+= {product-title} with Local Zones
 
-ROSA supports the use of AWS Local Zones, which are metropolis-centralized availability zones where customers can place latency-sensitive application workloads within a VPC. Local Zones are extensions of AWS Regions and are not enabled by default. When Local Zones are enabled and configured, the traffic is extended into the Local Zones for greater flexibility and lower latency. For more information, see "Configuring machine pools in Local Zones".
+{product-title} supports the use of AWS Local Zones, which are metropolis-centralized availability zones where customers can place latency-sensitive application workloads within a VPC. Local Zones are extensions of AWS Regions and are not enabled by default. When Local Zones are enabled and configured, the traffic is extended into the Local Zones for greater flexibility and lower latency. For more information, see "Configuring machine pools in Local Zones".
 
 The following diagram displays a ROSA cluster without traffic routed into a Local Zone.
 
-.ROSA cluster without traffic routed into Local Zones
-image::../images/354_OpenShift_ROSA_Local_Zones_0923_1.png[ROSA cluster without traffic routed into Local Zones]
+.{product-title} cluster without traffic routed into Local Zones
+image::../images/354_OpenShift_ROSA_Local_Zones_0923_1.png[{product-title} cluster without traffic routed into Local Zones]
 
-The following diagram displays a ROSA cluster with traffic routed into a Local Zone.
+The following diagram displays a {product-title} cluster with traffic routed into a Local Zone.
```

Comment:
In Classic, this seems to be showing as ROSA still for some reason:

<img width="908" height="318" alt="image" src="https://github.com/user-attachments/assets/43c373a4-f6c6-4100-938f-150a3c881021" />